### PR TITLE
need a code review, new to process.

### PR DESCRIPTION
**Pull request by jomivega**
Hello all, I had not done a pull request before, so none of my code had been seen by y'all. Would love the feedback and a code review!

### CODE REVIEW

File: Reminder/app/src/main/java/com/reminder/app/LoginActivity.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    private GoogleMap map;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.login);
+        context = this.getApplicationContext();
+        findViewById(R.id.sign_in_button).setOnClickListener(this);
+        map = ((MapFragment)getFragmentManager().findFragmentById(R.id.map)).getMap();
+        LatLng test = new LatLng(-33.867, 151.206);
```
Comment (samschaevitz) [inline Reminder/app/src/main/java/com/reminder/app/LoginActivity.java]: Add a "TODO(jordan): ..." to remove this test when you have real data?

File: Reminder/app/src/main/java/com/reminder/app/LoginActivity.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    private ConnectionResult mConnectionResult;
+    private GoogleMap map;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.login);
+        context = this.getApplicationContext();
+        findViewById(R.id.sign_in_button).setOnClickListener(this);
+        map = ((MapFragment)getFragmentManager().findFragmentById(R.id.map)).getMap();
```
Comment (samschaevitz) [inline Reminder/app/src/main/java/com/reminder/app/LoginActivity.java]: can you split this line into a few lines to make it easier to read?

File: Reminder/app/src/main/java/com/reminder/app/LoginActivity.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        LatLng test = new LatLng(-33.867, 151.206);
+        map.setMyLocationEnabled(true);
+        map.moveCamera(CameraUpdateFactory.newLatLngZoom(test, 13));
+        map.addMarker(new MarkerOptions()
```
Comment (samschaevitz) [inline Reminder/app/src/main/java/com/reminder/app/LoginActivity.java]: Maybe all of these map init calls can be broken out into a helper function?

File: Reminder/app/src/main/java/com/reminder/app/LoginActivity.java [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        }
+    }
+    public void onClick(View view) {
+        if (view.getId() == R.id.sign_in_button
+                && !mGoogleApiClient.isConnecting()) {
+            mSignInClicked = true;
+            resolveSignInError();
+        }
+    }
+    @Override
```
Comment (samschaevitz) [inline Reminder/app/src/main/java/com/reminder/app/LoginActivity.java:114]: Let's always do this (add @Override to other class methods, please). Keeps inheritance easier to reason about.

File: Reminder/app/src/main/java/com/reminder/app/LoginActivity.java [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    public void onConnected(Bundle connectionHint) {
+        mSignInClicked = false;
+        if (Plus.PeopleApi.getCurrentPerson(mGoogleApiClient) != null) {
+            currentPerson = Plus.PeopleApi.getCurrentPerson(mGoogleApiClient);
```
Comment (samschaevitz) [inline Reminder/app/src/main/java/com/reminder/app/LoginActivity.java]: Let's split the Person setup and the Email setup out into helpers for readability.

**Comment by samschaevitz** [comment]
Couple of comments; didn't really look at the XML files but so long as you add a comment saying it works (TESTED=...whatever you did to test this code...) in the commit, I'm fine with it.

**Comment by jomivega** [comment]
Cleaned up some of the code and worked on a standardized UI. Here are some images of the UI. Do you all like the profile picture in the navigation bar? If so, on the left or right side? Also, I changed the navigation icon to white, and positioned it. I changed the font of the items in the drawer as well. Let me know what you think! 
screenshot_2014-06-27-00-59-21
screenshot_2014-06-27-00-58-29

screenshot_2014-06-27-00-58-15

**Comment by samschaevitz** [comment]
I vote for right side; it feels more balanced. Digging the font and color schema! Maybe there could be sub menus in the left bar for "Upcoming" and "Done" reminders?

**Comment by jomivega** [comment]
Ooh yes I like that Idea. Should I work on that after I merge what I have here?

**Comment by samschaevitz** [comment]
SGTM (sounds good to me)

**Comment by jomivega** [comment]
Awesome. Just waiting for an okay from either Kylie or Jooeun.

**Comment by samschaevitz** [comment]
Can you call one of them out as a reviewer on this pull request? That way
it'll be clearer who needs to own the review.

**Comment by JooeunLim** [comment]
I like the right side as well, and the upcoming/done list idea too. Should
the add reminder page have the same green or a different color?

**Comment by jomivega** [comment]
What do you mean Jooeun? Like should you add the top bar and drawer? Or? Jooeun can you also be the reviewer for this pull request?

**Comment by JooeunLim** [comment]
No, I mean just the color scheme in general. I was thinking the add
reminder page  would just have a back button or something to go back home
and not have the sidebar. And yup!

**Comment by jomivega** [comment]
If you create the add reminder page using the same color scheme, it will be perfect. Contact me, or look through the colors.xml to get the value of the green. No need for a back button.

**Comment by jomivega** [comment]
In my latest commit, I have added Kylie's UI from the master branch. 
Need to talk you Kylie about the layout so that it works with the container/standardized UI I have set up. 

Also, for all that are interested, I have ran the go backend through golint and made changes accordingly. You can see the changes in this commit if you would like. 

Sam, let me know what you think about the backend code.

**Comment by jomivega** [comment]
Updated the UI and added kylie's UI to the standardized UI containter. Let me know your thoughts about the action bar and submenues.
screenshot_2014-06-27-18-23-41
screenshot_2014-06-27-18-39-12
screenshot_2014-06-27-18-39-18

**Comment by jomivega** [comment]
_Ignore the white spots_

**Comment by bumbledog** [comment]
Still not compiling correctly... Jooen, could you see if it runs on your computer?

**Comment by bumbledog** [comment]
Here's the error I keep getting.
2032-2032/com.reminder.app E/﹕ Can't open file for reading
This happens whenever I try to login to google plus on the app via an emulator.

**Comment by bumbledog** [comment]
I'm 90% sure it's a connection problem with the backend. Whenever I try to step through, I'm getting a lot of connection errors.

**Comment by bumbledog** [comment]
Oops, I did not mean to close this...

**Pull request closed without merging**